### PR TITLE
chore: 01_sick-notes.md

### PR DESCRIPTION
**Pull request by erabeuf**
## Checklist

- [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) guidelines 
- [ ] All relevant PR sections are populated, irrelevant ones are removed

### CODE REVIEW

File: src/regions/ca/info/faq/22_get-food.fr.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
-## Puis-je ou dois-je aller chercher de la nourriture?
+## Puis-je aller à l'épicerie ?
```
Comment (hollyorlando) [inline src/regions/ca/info/faq/22_get-food.fr.md:1]:
```suggestion
## Puis-je aller à l'épicerie?
```

File: src/regions/ca/info/faq/26_tested-work.fr.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## Mon employeur souhaite que je passe un test avant de retourner au travail car j'ai voyagé ces derniers jours. Que dois-je faire?
 
-Vous pouvez appeler la ligne de votre autorité de santé publique pour le COVID-19. Ils évalueront le risque et décideront si vous devez vous faire dépister. En attendant, assurez-vous de vous isoler.
+Vous devez contacter la ligne de votre Autorité de santé publique pour le COVID-19. Ils évalueront le risque et décideront si vous devez vous faire tester. En attendant, assurez-vous de vous isoler pendant 14 jours, à partir du jour de votre retour au Canada. Il en va de même pour les voyages vers des régions soumises à des interdictions de voyager régionales et interprovinciales qui sont en vigueur dans de nombreuses juridictions au Canada.
```
Comment (hollyorlando) [inline src/regions/ca/info/faq/26_tested-work.fr.md:3]:
```suggestion
Vous devez contacter la ligne de votre Autorité de Santé Publique pour la COVID-19. Ils évalueront le risque et décideront si vous devez vous faire tester. En attendant, assurez-vous de vous isoler pendant 14 jours, à partir du jour de votre retour au Canada. Il en va de même pour les voyages vers des régions soumises à des interdictions de voyager régionales et interprovinciales qui sont en vigueur dans de nombreuses juridictions au Canada.
```

File: src/regions/ca/info/faq/26_tested-work.fr.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
 
-Plus d'informations [ici](https://www.cnesst.gouv.qc.ca/salle-de-presse/Pages/coronavirus.aspx).
+Plus d'informations ici:
+
+- [La ressource fiable canadienne pour les mises à jour pour le COVID-19](https://fr.c19.ca/)
+- [Chloé pour la COVID-19](https://covid19.dialogue.co/#/)
```
Comment (hollyorlando) [inline src/regions/ca/info/faq/26_tested-work.fr.md:8]: It’s odd to reference our own website, this is done a few times in this PR

Comment (erabeuf) [reply src/regions/ca/info/faq/26_tested-work.fr.md:8]: C'est ce que j'avais dit à Marie-Pierre, mais c'est que la FAQ est envoyé directement aux patients et ils ne voient pas Chloe quand ils sont dans la FAQ.

Comment (hollyorlando) [reply src/regions/ca/info/faq/26_tested-work.fr.md:8]: Hmmm all good then :)

File: src/regions/ca/info/faq/01_sick-notes.fr.md [hunk 4]
```diff
@@ -1,9 +1,15 @@
-## La position de Dialogue sur les billets médicaux
+## Quelle est la position de Dialogue sur les notes médicales liées au COVID-19 ?
```
Comment (hollyorlando) [inline src/regions/ca/info/faq/01_sick-notes.fr.md:1]:
```suggestion
## Quelle est la position de Dialogue sur les notes médicales liées au COVID-19?
```

File: src/regions/ca/info/faq/03_test-referral.fr.md [hunk 5]
```diff
@@ -1,11 +1,9 @@
 ## Est-ce qu’un praticien de Dialogue peut référer pour un test du Coronavirus ? Est-ce qu’une référence peut-être donnée via la plateforme ou est-ce que le praticien doit évaluer physiquement le patient ?
```
Comment (hollyorlando) [inline src/regions/ca/info/faq/03_test-referral.fr.md:1]:
```suggestion
## Est-ce qu’un praticien de Dialogue peut référer pour un test du Coronavirus? Est-ce qu’une référence peut-être donnée via la plateforme ou est-ce que le praticien doit évaluer physiquement le patient?
```

File: src/regions/ca/info/faq/03_test-referral.md [hunk 6]
```diff
@@ -1,5 +1,9 @@
...
+To comply with requests from the federal and provincial governments, any suspected cases still must go through the provincial healthcare authority. Local health authorities have specific care paths patients need to follow as they centralize all cases in specific testing and treating locations. Dialogue can assess patients, but any suspected cases will be directed to the appropriate health authority. Patients will be provided with all the relevant information they need along with safety guidelines to avoid further spread of the virus
 
-We recommend that anyone who may require testing consults on the Dialogue platform, as we will be able to give the patients the most up-to-date and relevant advice for their location and situation. Test sites may vary depending on the patient’s situation and the degree of risk and illness.
+More information here :
```
Comment (hollyorlando) [inline src/regions/ca/info/faq/03_test-referral.md:5]:
```suggestion
More information here:
```

File: src/regions/ca/info/faq/11_family-isolating.fr.md [hunk 7]
```diff
@@ -1,7 +1,22 @@
-## Que faire si la famille ou les amis s'isolent?
+## Que faire si quelqu'un chez moi a le COVID-19?
 
-Toutes les recommandations se trouvent sur ces sites web:
+Le gouvernement du Canada a fourni [cette excellente page](https://www.canada.ca/fr/sante-publique/services/publications/maladies-et-affections/comment-prendre-soin-personne-atteinte-covid-19-maison-conseils-soignants.html) qui décrit toutes les procédures recommandées, les précautions, les produits de nettoyage nécessaires et plus encore.
 
-- [Gouvernement du Canada](https://www.canada.ca/fr/sante-publique/services/publications/maladies-et-affections/covid-19-comment-isoler-chez-soi.html)
-- [CDC](https://www.cdc.gov/coronavirus/2019-ncov/prepare/get-your-household-ready-for-COVID-19.html) (Anglais seulement)
-- [CDC](https://www.cdc.gov/coronavirus/2019-ncov/about/steps-when-sick.html?mod=article_inline) (Anglais seulement)
+Si quelqu'un chez vous a le COVID-19, il y a d'autres choses que vous pouvez faire pour vous protéger et protéger les autres :
```
Comment (hollyorlando) [inline src/regions/ca/info/faq/11_family-isolating.fr.md:5]:
```suggestion
Si quelqu'un chez vous a le COVID-19, il y a d'autres choses que vous pouvez faire pour vous protéger et protéger les autres:
```

File: src/regions/ca/info/faq/11_family-isolating.fr.md [hunk 8]
```diff
@@ -1,7 +1,22 @@
...
+
+- **Tenir la personne malade à l'écart des autres** - La personne malade doit rester dans une chambre séparée et utiliser une autre salle de bain si possible. Elle doit également manger dans sa propre chambre.
+
+Les experts recommandent également que la personne reste à l'écart des animaux domestiques dans la maison jusqu'à ce qu'ils aillent mieux.
+
+- **Faites-leur couvrir leur visage** - La personne malade doit se couvrir le nez et la bouche avec un masque en tissu lorsqu'elle se trouve dans la même pièce que d'autres personnes. Si elle ne peut pas utiliser de masque, vous pouvez l'aider à se protéger en vous couvrant le visage lorsque vous êtes dans la pièce avec elle.
+
+- **Se laver les mains** - Se laver souvent les mains avec de l'eau et du savon (voir ci-dessus).
+
+- **Nettoyez-vous souvent** - Voici quelques éléments spécifiques qui peuvent vous aider :
```
Comment (hollyorlando) [inline src/regions/ca/info/faq/11_family-isolating.fr.md:15]:
```suggestion
- **Nettoyez-vous souvent** - Voici quelques éléments spécifiques qui peuvent vous aider:
```

**Comment by hollyorlando** [comment]
Requested a few small changes but overall looks great! 🚀

**Pull request merged**